server/server.cc

```diff
@@ -101,7 +101,7 @@ int main(int argc, char **argv) {
   Log<Output2FILE>::reportingLevel() = LOG_DEBUG;
   FILE* log_fd = fopen(ndnfs::server::logging_path.c_str(), "w" );
   if (ndnfs::server::logging_path == "" || log_fd == NULL) {
-	Output2FILE::stream() = stdout;
+	  Output2FILE::stream() = stdout;
```

Comment:
what changed here? is this correct indentation per code style?

Reply:
Well, my mistakes. The original codes mixes TAB and spaces. I want to use spaces only but it seems I didn't change all of them.